src/com/cft/contactmerge/AddressMatchSelector.java

```diff
@@ -0,0 +1,52 @@
+package com.cft.contactmerge;
+
+import java.util.ArrayList;
+
+public abstract class AddressMatchSelector {
+    /*******************************************************************************************************************
+     *********************************************** Street Address Only ***********************************************
+     *******************************************************************************************************************/
+    public static AnswerType getStreetAddressMatchResultDefaultComparisons(String stAddress1, String stAddress2){
```

Comment:
You indicated that you would like feedback on the design. Generally, it is frowned on to have a bunch of static methods as you are basically reverting back to functional programming. If you take a look at the high level class diagram I created, and the stubs you reviewed in the contactmerge.contact package, my proposal is that we have an object for each of the main components for a contact. There would be Name, Address, Phone Number, and Email. These objects can be made of properties. Name is made up of FirstName and LastName at the moment. Address is made up of Street Address, City, State, Country, and Zip. Each of these objects (Address, Street Address, City, State, etc) has an isMatch() method. The base/parent version of a Property would simple have a isMatch() method that would return values.equalsIgnoreCase(value2) ? AnswerType.Yes : AnswerType.No; Special properties like StreetAddress, Address, FirstName, Name, etc would override isMatch() to include the logic in your static methods. There would still be some static methods but they would focus on basic string manipulation.